mkdocs/pages/en/devbook/accounts/create-from-mnemonic.md

```diff
@@ -0,0 +1,118 @@
+
+{% import 'tutorial.jinja2' as tutorial with context %}
+
+{{ tutorial.code_full('devbook/accounts/create-from-mnemonic', ['py', 'js']) }}
+
+## Code Explanation
+
+### Initializing the Facade
+
+{{ tutorial.code_snippet(['py:5:6', 'js:5:6']) }}
+
+The <dy:SymbolFacade> provides access to Symbol's cryptographic operations and network utilities.
+It is initialized with a network name (`testnet` or `mainnet`) to ensure that network-specific values,
+such as <address:>, are generated correctly.
+
+### Defining a Mnemonic
+
+{{ tutorial.code_snippet(['py:8:16', 'js:8:17']) }}
+
+The example checks for an existing mnemonic in the `MNEMONIC` environment variable.
+If the variable is set, the mnemonic is loaded from it.
+Otherwise, a new random mnemonic is generated using <dy:Bip32.random>.
+
+Symbol uses the [BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) standard, which represents
+mnemonics as 24 English words selected from a standardized word list.
+These words encode entropy that serves as the foundation for deriving private keys.
+
+!!! warning "Store your mnemonic securely"
+    The mnemonic can be used to regenerate all derived accounts and private keys.
+    Anyone with access to it can control your accounts, and losing it means losing access permanently.
+
+    Never share your mnemonic with anyone, and always store it in a secure location.
+
+### Deriving the Root Node
+
+{{ tutorial.code_snippet(['py:18:23', 'js:19:24']) }}
+
+After defining the mnemonic, <dy:Bip32.fromMnemonic> converts the mnemonic and a password into a root node,
+which serves as the starting point for deriving child accounts.
+
+The password (sometimes called a "25th word") is an optional string that extends the mnemonic seed.
+It can be left empty or set to any value.
+When used, it adds another layer of security. Different passwords with the same mnemonic produce completely different
+accounts.
+
+!!! note "Password security"
+    The password is part of the account derivation.
+    Both the mnemonic and password are required to regenerate the accounts.
+    If you lose or forget either one, you lose access to all derived accounts.
+
+In this example, the password is loaded from the `PASSWORD` environment variable.
+If not set, the snippet defaults to `correcthorsebatterystaple` for demonstration purposes only.
+
+### Deriving the Child Account
+
+{{ tutorial.code_snippet(['py:25:27', 'js:26:28']) }}
+
+The root node can derive multiple child accounts using a hierarchical path.
+This allows a single mnemonic to manage many accounts while keeping them cryptographically isolated.
+
+<dy:SymbolFacade.bip32Path> generates the derivation path for a specific account index.
+In this example, the account at index `0` is derived, but you can derive additional accounts by using different
+indices (e.g., `1`, `2`, `3`).
+
+<dy:Bip32Node.derivePath> follows the specified path from the root node to produce a child node that represents
+the account.
```

Comment:
This explanation is a bit obscure. What are hierarchical paths and derivation paths? So I can derive child nodes from other child nodes and build a tree? Why would I want that?

HOWEVER, I don't think many people care about that, and we already provided a link to the BIP39 standard, so this section is fine by me.

In any case, if we wanted to talk more about this topic, we'd need to do it in the Textbook.